arangod/Sharding/ShardDistributionReporter.cpp

```diff
@@ -339,71 +343,73 @@ void ShardDistributionReporter::helperDistributionForDatabase(
             }
 
             // Ask them
-            std::unordered_map<std::string, std::string> headers;
+            std::vector<network::FutureRes> futures;
             for (auto const& server : serversToAsk) {
```

Comment:
we could add `futures.reserve(serversToAsk.size());` as a micro optimization here.